charts/jenkins/jobs/simflofy/nuxeo-simflofy.groovy.gotmpl

```diff
@@ -23,7 +23,7 @@ multibranchPipelineJob('simflofy/nuxeo-simflofy') {
             // Filter by name (with regular expression)
             headRegexFilter {
               // Regular expression
-              regex('master|2021|PR-.*')
+              regex('master|2021|2023|PR-.*')
```

Comment:
Should this change be removed? There's no 2023 branch on nuxeo-simflofy repository.